### DISCUSSION TITLE
"avoid highways" (german Autobahn) vs "prohibit highways"

### DISCUSSION THREAD

**Discussion by ThomasMeister123** [osmandapp/OsmAnd#13521] [Q&A]
How to create a profile, that PROHITS highways (german Autobahn), because some vehicles are not allowed there. With "avoid highways" the route pissibly goes via a highway....

"avoid highways" only lowers the priority, right?

**Comment by vshcherb** [comment]
You can create custom routing.xml , we moved from prohibit to avoid because we need to support situation when car need to move away from highway if the road accidentally starts there. 
https://docs.osmand.net/en/main@latest/development/osmand-file-formats/osmand-routing-xml
